cluster-install/install.sh

```diff
@@ -2250,28 +2371,33 @@ step_monitor_deployment_progress() {
     if components_include_connector; then
         if is_dry_run; then
             logdebug "skip monitoring connector components because it's a dry run"
-        elif ! wait_for_deployment_running "operator-controller-manager" "$connector_operator_ns" "1m"; then
+        elif ! wait_for_deployment_running "operator-controller-manager" "$connector_operator_ns" "3"; then
```

Comment:
With the change to the `wait_for_deployment_running`, we were no longer passing over the timeout value to kubectl for us to parse, so `1m` -> `1` and we just assume the unit is minutes every time. Don't think we really need the granularity of seconds or the scale of hours anyhow.